spec.bs

```diff
@@ -575,11 +575,24 @@ Moreover, each {{SharedStorageWorklet}}'s [=global scopes|list of global scopes=
         1. If |filteringIdMaxBytes| is not the [=default filtering ID max bytes=] or
             |contextId| is not null, return a new {{DOMException}} with name
             "`DataError`".
+    1. Let |maxContributions| be null.
+    1. If
+        |privateAggregationConfig|["{{SharedStoragePrivateAggregationConfig/maxContributions}}"]
+        [=map/exists=], set |maxContributions| to
+        |privateAggregationConfig|["{{SharedStoragePrivateAggregationConfig/maxContributions}}"].
+    1. If |maxContributions| is not a positive integer, set |maxContributions| to null.
```

Comment:
I think negative numbers will cause an error due to the `[EnforceRange]`, so maybe we treat zero similarly for consistency? (Also to 'fail fast')

Reply:
Ah yes, EnforceRange generates an exception. I'll just return a DataError here when the value is zero.